### ISSUE TITLE
Basic javascript tests deletes the configuration file

### ISSUE THREAD

**Issue by thomasdangl** [se2p/programming-styles-ss-2020-public-tests#47]
While #46 fixed #45, it also introduced a new bug.
The test scenario in basic-tests deletes the configuration file before and after each test, but never restores it.
This violates the specification but more importantly it breaks all file tests, if the basic tests are executed first.
```
 1) File Tests
       should produce the expected output (absolute_mayhem):
     Error: Configuration file not found.
```
I think I can do a pull request for this stuff later on if I find some time.

**Comment by alessiogambi** [comment]
Uhm. Maybe I forgot to commit that, but I also updated files tests. File tests create the config files before executing each test. If I remember correctly your code looks for the config file under `. /config.ini` instead of `__dirname+config.ini` try to change that and see if tests pass

**Comment by thomasdangl** [comment]
Oh sorry, I did not notice that file changed aswell.
However, even when using the workaround you mentioned, the tests fail for me.
I think what's happening here is that basic-tests.js deletes the files inside of pahHome, but file-tests.js writes to the test directory instead.
Let me quickly trace the file system calls to confirm this suspicion.

Edit: Yup, this was the issue at hand. I'll create a pull request.

**Comment by thomasdangl** [comment]
Nevermind, the issue is already fixed with fbe5962270eb92f9e16071f2f8f6b891de766c62.
I tested with 6445ba41222389878c34a0984a0ac8662a135fda (the merge commit) and did not notice the next patch.
Sorry for any inconvenience caused.
